average_calculation.py

```diff
@@ -15,13 +14,28 @@
 # the rest of them is optional
 
 api = RiaAPI()
+myCarAveragePrice = RiaAverageCarPrice(
+    category='Легковые',
+    mark='Renault',
+    model='Scenic',
+    # bodystyle=None,
+    years=[2006, ],
+    # state=None,
+    gears=['Механика'],
+    opts=['ABS'],
+    mileage=[10, 200],
+)
+
+pprint(myCarAveragePrice.get_average())
 
-results = api.get_average(category='Легковые',
-                          mark='Renault',
-                          model='Scenic',
-                          bodystyle=None,
-                          years=[2014, ],
-                          state=None
-                          )
+"""
```

Comment:
The triple-quote strings are used for docstings, and it looks unusual (=not good) here.
If this is related to the module, it should be at the top (similarly, we shouldn't have such triple-quoted strings in the middle of the function, only at the top, below the `def`).

In other places we can use regular comments starting with `#`.